### ISSUE TITLE
Doubts Stratification regression problems

### ISSUE THREAD

**Issue by javierperezcastan** [pycaret/pycaret#911]
Hi, 
I wonder about when we apply stratify in a regression problem. How the model do that? It calculate percentiles or quantiles?

I have tried also to introduce an ad-hoc column with the classes that I want to consider. However, after applying the set up method this column will be placed in the data to include in the model. How can I remove it?

Best regards
Javier

**Comment by Yard1** [comment]
We use sklearn's `train_test_split()` to perform stratification. Please refer to its documentation. I believe that it only accepts integer labels in the columns to stratify by.

Something like this should be possible to stratify by an adhoc column - if it doesn't work, let me know:
```
setup(data, target="target", data_split_stratify=["adhoc"], ignore_features=["adhoc"])
```

**Comment by javierperezcastan** [comment]
Perfect, it works¡¡
